Jenkinsfile

```diff
@@ -62,26 +65,54 @@ pipeline {
         }
     }
     post {
-        always {
-            sh '$WORKSPACE/trace.sh'
-            junit allowEmptyResults: true, healthScaleFactor: 1.0, testResults: '**/target/*-reports/TEST-*.xml'
-            findbugs canComputeNew: false, defaultEncoding: '', excludePattern: '', healthy: '', includePattern: '', pattern: '**/spotbugsXml.xml', unHealthy: ''
-            checkstyle canComputeNew: false, defaultEncoding: '', healthy: '', pattern: '**/checkstyle-result.xml', unHealthy: ''
-            archiveArtifacts allowEmptyArchive: true, artifacts: '**/*.maven.log, **/target/*.log,**/target/testStatusListener*' + additionalArtifactsToArchive, excludes: '**/target/checkstyle.log' + additionalExcludedArtifacts, fingerprint: false, defaultExcludes: true, caseSensitive: true
+        fixed {
+            script {
+                mailer.sendEmail_fixedPR(pr_type)
+            }
+        }
+        aborted {
+            script {
+                mailer.sendEmail_abortedPR(pr_type)
+            }
         }
         failure {
+            sh '$WORKSPACE/trace.sh'
+            script {
+                mailer.sendEmail_failedPR(pr_type)
```

Comment:
every `mailer.sendEmail*` method should be prepared for null values, otherwise in expressions like this
`"${pr_type} #$ghprbPullId of $ghprbGhRepository: $ghprbPullTitle failed"` the subject will be
`null #X of Y: WhateverTitle failed` which is not user friendly